client/src/components/EventView/index.js

```diff
@@ -1,7 +1,174 @@
 
-const EventView = () => {
-  return <div>Event View</div>;
+// const testEvent = {
+//     eventID: "testID",
+
+// }
+
+const CenterHeading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const CenterHeadingLight = styled.p`
+    font-size: calc(7px + 2vmin);
+    color: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Heading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+`;
+
+const DisplayStle = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+    width: 70%;
+    height: 100%;
+    padding: 10% 15%;
+`;
+
+const InviteeRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  //align-items: center;
+  align-items:flex-start;
+  //justify-content: space-around;
+`;
+
+const Main = () => {
+  const [apiStatus, setApiStatus] = useState({
+    loading: false,
+    error: false,
+    message: false
+  });
+
+    const [event, setEvent] = useState({
+        eventName: "testName"
+
+    });
+    useEffect(() => {
+      loadPageData();
+      // eslint-disable-next-line
+    }, []);
+
+    //not sure how to implement this
+    // useEffect(() => {
+    //     const params = new URLSearchParams(event.location.search);
+    //     let eventId = params.get("id");
+    //     if (eventId) {
+    //     loadPageData(eventId);
+    //     } else {
+    //     //redirect to home page since there isn't an eventId
+    //     }
+    //     // eslint-disable-next-line
+    //     }, [location.search]);
+    //please help Morgan!!
+
+  const loadPageData = async () => {
+    setApiStatus({ ...apiStatus, loading: true, error: false });
+    const newApiStatus = { ...apiStatus };
+    try {
+        let eventId = "12346"
+      //const results = await ClientService.events();
+      const results = await ClientService.event(eventId);
+      //TODO: call endpoint for invites
+      if (results.success) {
+        //prepareData(results.events);
+        
+        setEvent(results.event);
+      } else {
+        // default message just in case
+        
+        newApiStatus.error = true;
+        newApiStatus.message = results.message;
+      }
+    } catch (error) {
+      newApiStatus.error = true;
+      newApiStatus.message = error.message;
+    }
+    newApiStatus.loading = false;
+
+    setApiStatus(newApiStatus);
+  };
+
+//const Main = () => {
+  return (
+    <div>
+      <AppBar color={eventPink} name = {event.eventName} date = {event.date}>
+      </AppBar>
+      <div>
+      {apiStatus.loading ? (
+        <CenteredLoadingSpinner
+          size={150}
+          color={theme1}
+          loading={apiStatus.loading}
+        />
+      ) : (
+        <Stack gapSize={spacing64}>
+          <DisplayStle>
```

Comment:
Stack will space out all of the child items inside of it, and by default it is left-aligned. Right now you only have one child element, so that's defeating the purpose of the Stack. Try removing DisplayStyle and then you'll want to group the different pieces of your code in divs. For example, the entire description part will be wrapped in its own div, and the location part will be wrapped in its own div.